### DISCUSSION TITLE
页面刷新时下载该页面

### DISCUSSION THREAD

**Discussion by HanBiWolf** [volantis-x/hexo-theme-volantis#805] [Q&A] [answered]
在访问页面或者刷新网页时
会调用下载把一个文件下下来
然后用记事本打开看到是生成的index.html文件(下载时是以路径命名)
![image](https://user-images.githubusercontent.com/101497620/187120817-86d3f37c-9feb-415c-936c-75dba87a9075.png)
然后因为这个导致giscus登录回调的时候没法成功,页面也刷新不了

有大佬们知道怎么解决吗 :(

**Comment by HanBiWolf** [comment]
另外生成的文件是放在仓库的一个分支,然后由cloudflare page从这个分支进行部署
所以cdn的话是全站都在cloudflare上了

**Comment by MHuiG** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/38281581/187341249-846410cd-9294-4acc-9c5d-47776c9cd2d0.png)

这个标头一般是 `content-type: text/html; charset=utf-8`

**Reply by MHuiG** [reply]
content-type: application/octet-stream 是流式下载文件的标头

**Reply by MHuiG** [reply]
https://github.com/HanBiWolf/HBW-Blog-Hexo-Volantis/tree/sitepreview/posts

这个文件为什么没有扩展名？

**Reply by HanBiWolf** [reply]
好吧貌似我abbrlink插件配置可能有点毛病(
我得改改链接设置,改完看看(

**Reply by MHuiG** [reply]
permalink: posts/:abbrlink  => permalink: posts/:abbrlink/

**Reply by HanBiWolf** [reply]
刚刚改好了,我等Action部署一下先

**Reply by HanBiWolf** [reply]
dev那边改了没问题,我看看部署到cf上面有没有问题先

**Reply by HanBiWolf** [reply]
好了谢谢:P
问题解决了hh
(刚刚CF不知道是不是抽风了一直部署不上去,现在看到可以了)
